connect-examples/v2/java_payment/src/main/java/com/squareup/connectexamples/ecommerce/Main.java

```diff
@@ -78,35 +74,41 @@ public static void main(String[] args) throws Exception {
     private String mustLoadEnvironmentVariable(String name) {
         String value = System.getenv(name);
         if (value == null || value.length() == 0) {
-            throw new IllegalStateException(
-                String.format("The %s environment variable must be set", name));
+            throw new IllegalStateException(String.format("The %s environment variable must be set", name));
         }
 
         return value;
     }
 
     @RequestMapping("/")
     String index(Map<String, Object> model) throws ApiException {
-        model.put("paymentFormUrl", squareEnvironment.equals("sandbox") ?
-            "https://js.squareupsandbox.com/v2/paymentform" : "https://js.squareup.com/v2/paymentform");
+        model.put("paymentFormUrl",
+                squareEnvironment.equals("sandbox") ? "https://js.squareupsandbox.com/v2/paymentform"
```

Comment:
is  `squareClient` tell the environment information? Wondering if comparing strings can be replaced by environment check?

Reply:
I'm not too sure what your question is here. The string comparison is of an environment variable.

Reply:
Never mind, this is fine.